server.py

```
@@ -168,15 +202,38 @@ def receive_packet(packet, interface=interface):
+                    if rx_delta > WATCHDOG_TIMEOUT:
+                        logging.warning(f"No packets received for {int(rx_delta)}s. Forcing reconnect.")
                         should_sleep = True
                         break
+                    elif rx_delta > KEEPALIVE_INTERVAL and (now - last_keepalive_sent) > KEEPALIVE_INTERVAL:
+                        # Generate keepalive traffic to maintain connection on quiet meshes
+                        # We do a safe request like getting node info to trigger activity
+                        try:
+                            logging.debug("Mesh quiet, generating keepalive traffic...")
+                            interface.getNode(interface.myInfo.my_node_id)
+                            last_keepalive_sent = now
+                        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)